profile/README.md

```diff
@@ -7,6 +7,18 @@ To check out all the series of SoCs from Espressif, please visit our [ESP Produc
 
 Below you can find a selection of Espressif’s open-source projects. Our full repository list can be found [here](https://github.com/orgs/espressif/repositories).
 
+## Project Highlight
+
+<div align="center">
+    <img src="profile/esp_registry_logo.png">
+</div>
+
+###  **ESP Registry**
+
+The **ESP Registry** is a tool that downloads dependencies for any ESP-IDF CMake project. It makes sure that the right versions of all components required for a successful build of your project are in place. The download happens automatically during a run of CMake.
+
+Find the most exciting ESP-IDF components on [ESP Registry](https://components.espressif.com/)!
+If you want to contribute to this project, you can do so at [IDF Component Manager repository](https://github.com/espressif/idf-component-manager).
```

Comment:
I'm not sure if a lot of meaningful contributions can be done inside idf-component-manager repo, it's mostly the tooling part. How about we recommend the users to publish their components in the Registry, instead?

Reply:
Good point, I was mainly thinking about the feedback which should be shared in the idf-component-manager repository.